data.sql

```diff
@@ -33,3 +33,89 @@ UPDATE animals SET owner_id=2 WHERE name IN ('Pikachu', 'Gabumon');
 UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
 UPDATE animals SET owner_id=4 WHERE name IN ('Charmander','Squirtle','Blossom');
 UPDATE animals SET owner_id=5 WHERE name IN ('Angemon','Boarmon');
+
+-- vets data
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('William Tatcher', 45, '2000-4-23');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Maisy Smith', 26, '2019-1-17');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-5-4');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Jack Harkness', 38, '2008-6-8');
+
+-- specialties data: 
+INSERT INTO specializations (vets_id, species_id) VALUES (1, 1);
+INSERT INTO specializations (vets_id, species_id) VALUES (3, 1);
+INSERT INTO specializations (vets_id, species_id) VALUES (4, 2);
+
```

Comment:
- [ ] The requirements say that `Vet Stephanie Mendez is specialized in Digimon and Pokemon` but you implement only for the `first species` , Kindly improve that